common/cluster/metadata.go

```diff
@@ -49,8 +49,11 @@ type (
 		ClusterNameForFailoverVersion(failoverVersion int64) string
 		// GetAllClientAddress return the frontend address for each cluster name
 		GetAllClientAddress() map[string]config.Address
-		// GetDeploymentGroup returns the deployment group of cluster
-		GetDeploymentGroup() string
+
+		// IsArchivalEnabled whether archival is enabled
+		IsArchivalEnabled() bool
```

Comment:
see comment about this feature flag above

Reply:
spoke offline about this one.